superset-frontend/src/explore/components/DatasourcePanel/index.tsx

```diff
@@ -373,7 +373,9 @@ export default function DataSourcePanel({
                     <span
                       role="button"
                       tabIndex={0}
-                      onClick={() => setShowSaveDatasetModal(true)}
+                      onClick={() => {
```

Comment:
Isn't the Add Dataset page only for adding physical datasets, and this modal is for creating a virtual dataset from a query?  If so, should it be left as-is?

Reply:
Is this spot only for virtual datasets or is it also for physical? We currently don't have plans to add a virtual dataset from the new flow, the user can create them through the SQL Lab route by creating a chart from a query, then creating a dataset off of that in explore.

Reply:
I saw your other comments about virtual dataset creation, are those all solely for virtual dataset creation or is it also for physical?

Reply:
Confirmed: virtual only

Reply:
Fixed in [`this commit`](https://github.com/apache/superset/pull/22610/commits/d2903e307228270c40270088b7a83d64ae13e545).